webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -5175,6 +5175,8 @@ const firstVisibleMsg = {
 
                 $applozic(d).on('click', '#km-talk-to-human', function (e) {
                     e.preventDefault();
+                    var $button = $applozic(this);
```

Comment:
@jaitens Let's use the javascript instead of the jQuery because we are trying to remove the jquery

Reply:
done